fvwm/expand.c

```diff
@@ -391,6 +394,34 @@ static signed int expand_vars_extended(
 			return -1;
 		}
 		if (*(rest + n) != 0)
+		{
+			/* Check for .lightenN or .darkenN */
+			csadj = 101;
+			l = -1;
+			if (sscanf(rest + n, ".lighten%d%n", &l, &nn) && l >= 0 && l <= 100)
+			{
+				csadj = l;
+			}
+			if (sscanf(rest + n, ".darken%d%n", &l, &nn) && l >= 0 && l <= 100)
+			{
+				csadj = -l;
+			}
+			/* Check for .hash */
+			if (strcmp(rest + n + nn,  ".hash") == 0)
```

Comment:
I would suggest the following:

```suggestion
			if (StrEquals(rest + n + nn,  ".hash"))
```

`StrEquals()` doesn't care about case, so I can use: `.HaSh` if I wanted, but I can't with plain `strcmp()`.  I know I've been guilty of this myself in the past.  Need to change this.

You could also offset the string comparison to elide the `.`, i.e.:  `rest + n + nn + 1`, but it's probably not worth it.